Controllers/FrontController.php

```diff
@@ -0,0 +1,156 @@
+<?php
+declare(strict_types=1);
+
+namespace Controllers;
+
+use Interfaces\IDataManagement;
+use Models\ProjectModels\Logger;
+use Models\ProjectModels\DataRegistry;
+use Models\ProjectModels\Server;
+use Models\ProjectModels\Session;
+use Models\ProjectModels\Post;
+use Models\ProjectModels\File;
+use Models\ProjectModels\Config;
+
+/**
+ * Main class for routing
+ *
+ * @package Controllers
+ */
+class FrontController
+{
+    /**
+     * Given controller
+     */
+    protected string $controller;
+
+    /**
+     * Given action
+     */
+    protected string $action;
+
+    /**
+     * Given params values
```

Comment:
I don't think such a property comments are really useful. Don't you think it just increases code lines in your class declaration, do you?